### PR TITLE
feat: update console wallet notifications

### PR DESCRIPTION
**Pull request by philipr-za**
Description
---
This PR creates console wallet notifications for transaction events and also adds a hotkey to clear notifications.

A bug was also spotted in the way that Faux transactions were validated that resulted in the last Faux transaction confirmed event to be emitted after each validation. The validation was updated to only emit this event if the transaction was previously not confirmed.

### CODE REVIEW

File: applications/tari_console_wallet/src/ui/components/notification_tab.rs [hunk 1]
```diff
@@ -4,9 +4,6 @@
 // are cleared.
 // Currently notifications are only added from the wallet_event_monitor which has
 // add_notification method.
-// TODO: auto delete old notifications. #LOGGED
```
Comment (philipr-za) [inline applications/tari_console_wallet/src/ui/components/notification_tab.rs:7]: This TODO is handled by the ability to manually clear the notifications, I don't think its ideal to do it automatically and notifications are not persisted so the manual option seems adequate.

File: applications/tari_console_wallet/src/ui/components/notification_tab.rs [hunk 2]
```diff
@@ -4,9 +4,6 @@
...
-// TODO: add interaction with the notifications, e.g. if I have a pending transaction
```
Comment (philipr-za) [inline applications/tari_console_wallet/src/ui/components/notification_tab.rs:8]: I don't think this TODO is worthwhile. All the events come from a single pane so there is only one place to go which is one right arrow press away. There is also no mouse interactivity in the console wallet so the concept of "clicking" doesn't exist.

**Review by delta1 — APPROVED** [review]
Very nice

**Pull request merged**